src/components/pages/HomePage.js

```diff
@@ -43,12 +43,16 @@ export default class HomePage extends React.PureComponent {
           <div className="content-wrapper">
             <h2>Welcome to Delivery Console!</h2>
             <h3>This is your one stop shop for shipping things to the browser.</h3>
-            <div className="cta">
-              <Button type="primary" size="large">
-                Let's Get Started
-              </Button>
-              <Button size="large">Another Button</Button>
-            </div>
+
+            {/* TODO: Unhide these buttons once we have appropriate content for them. */}
+            {false && (
+              <div className="cta">
+                <Button type="primary" size="large">
+                  Let's Get Started
+                </Button>
+                <Button size="large">Another Button</Button>
+              </div>
+            )}
```

Comment:
why the `false &&` trick and not simply using {/* */}?

Reply:
good call